_data/translations/pt-br.yml

```diff
@@ -0,0 +1,31 @@
+homeUrl: /pt-br/
+
+title: Cartas de Sarajevo
+
+url: "https://lettersfromsarajevo.com/pt-br/"
+
+metaDescription: >-
+  O começo da Guerra da Bósnia.
+  Kosta, um garoto de dez anos, foge de Sarajevo com sua mãe e irmão mais novo. Seu pai fica na cidade, sitiada e bombardeada, sobrevivendo a tempos difíceis com outros sarajevanos. Apesar de tudo, ele escreve cartas divertidas para seus filhos, tentando encorajá-los e animá-los.
+  Dez anos depois, usando as cartas originais escritas por seu pai, Kosta escreve a história de sua infância.
+
+metaImage: letters-from-sarajevo.jpg # TODO
+
+firstLetterLink: "/pt-br/primeira-carta"
+firstLetterText: "Primeira carta"
+
+prevLinkText: Carta anterior
+nextLinkText: Próxima carta
+
+homeLinkText: Home page
```

Comment:
Should we translate this, or is this common in Brasil?

Reply:
We often use "Home page" here in Brasil, it could be "Página inicial" as well, both work.